fv3core/stencils/fvtp2d.py

```diff
@@ -117,6 +119,31 @@ def __init__(self, namelist, hord):
         self.y_piecewise_parabolic_outer = YPiecewiseParabolic(
             namelist, ord_outer, self.grid.is_, self.grid.ie
         )
+        origin = self.grid.full_origin()
+        domain = self.grid.domain_shape_full()
+        ax_offsets = axis_offsets(spec.grid, origin, domain)
+        self._copy_full_domain = StencilWrapper(
+            func=copy_defn, origin=origin, domain=domain
+        )
```

Comment:
this also needs to extend to domain=self.grid.domain_shape_full(add=(0, 0, 1)), then everything will validate! woo nice job!

Reply:
thanks a lot!